src/lib.rs

```diff
@@ -143,6 +143,28 @@ impl<S> UniCase<S> {
         UniCase(Encoding::Unicode(Unicode(s)))
     }
 
+    /// Creates a new `UniCase` which performs only ASCII case folding.
+    #[cfg(__unicase__const_fns)]
+    pub const fn ascii(s: S) -> UniCase<S> {
+        UniCase(Encoding::Unicode(Unicode(s)))
```

Comment:
This probably should be the `Ascii` variant 😅

Reply:
Oh, duh. I copied this from the `unicode()` constructor if you couldn't tell.

Reply:
@seanmonstar fixed and added tests